src/Lykke.Service.BlockchainSettings.Services/BlockchainValidationService.cs

```diff
@@ -33,7 +33,7 @@ public async Task<bool> ValidateHotwalletAsync(string apiUrl, string type, strin
             }
             catch (Exception e)
             {
-                _log.Info(nameof(ValidateHotwalletAsync), $"{apiUrl}, {type}, {address}", "Could not check validity");
+                _log.Info(nameof(ValidateHotwalletAsync), "Could not check validity", $"{apiUrl}, {type}, {address}", e);
```

Comment:
Your message is passed to the ```context``` parameter. This overload is used https://github.com/LykkeCity/CommonDotNetLibraries/blob/master/src/Lykke.Common/Log/MicrosoftLoggingBasedLogExtensions.cs#L247. Specify ```message``` and ```exception``` parameter names explicitly.